src/wagtail_factories/blocks.py

```diff
@@ -17,6 +18,12 @@
 ]
 
 
```

Comment:
Maybe it can be done in one line rather than reassigning value to a variable that should be a constant variable.
`FACTORY_BOY_VERSION = tuple(int(x) for x in factory.__version__.split("."))`

Reply:
Hi @yohanlebret! Yup, it can be done, but should we? :)

This is actually a common refactor I do, where I split a multi-step list comprehension into new lines for each transform. The upside with this is readability (each line does its own thing) and maintenance (in VC a change in a transform step does not flag all steps as changed). The downside is of course you reassigning variables, it is a bit slower (nothing that effects this library though) and it occupies vertical space.

Here we interact with factory-boy and my idea with using one-line per transform instead of a list comprehension was to have a high degree of readability and to make any future changes to this portion of the code clear, since we don't own the `factory.__version__` value.

Does my reasoning change your mind @yohanlebret? If not, lets go with a list comprehension since this is just a style choice in the end and my FP (one transform per-line) approach here is a more unusual pattern in python.

Reply:
yes, it makes sense, your way is more readable.

Reply:
Awesome, in that case I'm resolving this conversation :)